src/main/webapp/entries.js

```diff
@@ -42,11 +40,12 @@ module.exports = {
   "sequencing-runs": "./resources/js/pages/sequencing-runs/index.js",
   groups: "./resources/js/pages/UserGroupsPage",
   "project-ncbi-exports": "./resources/js/pages/projects/ncbi-export",
-  "project-ncbi-export": "./resources/js/pages/projects/ncbi-export/ncbi-export.js",
+  "project-ncbi-export":
```

Comment:
I think this can be removed, it is reference in a html file but I cannot find that html file loaded anywhere. Can you check to see if this can be removed.